node/pkg/logscribeconsumer/app.go

```diff
@@ -131,29 +134,33 @@ func isLogLevelValid(entry map[string]any) bool {
 	return true
 }
 
-func bulkCopyLogEntries(ctx context.Context, logEntries []map[string]any) error {
-	bulkCopyEntries := [][]any{}
+func (a *App) bulkCopyLogEntries(logEntries []map[string]any) error {
```

Comment:
I think the function name should be changed to something else, it doesn't bulkCopy into pgsql directly anymore